tests/main.rs

```diff
@@ -37,6 +37,12 @@ const W9J_HASHES: &[(i32, &str)] = &[
     (10, "479c0a020eaceff5539e2dda2200c1ab"), // 5898846
 ];
 
+const W12J_SECOND_HASHES: &[(i32, &str)] = &[
+    (0, "1133f539027171d60d87a58f56abf094"),
+    (2, "f6c1c20b5613ba1c128783dd4d3e8ff5"),
+    (4, "f6c1c20b5613ba1c128783dd4d3e8ff5"),
```

Comment:
Is this the correct hash?

Note that the tests are failing: https://travis-ci.org/github/Rufflewind/wigner-symbols-rs/builds/773396798